app/src/widgets/list.ts

```diff
@@ -75,6 +75,15 @@ export default {
     initialOptions: {},
     initialMeta: {}
   },
+  todoist: {
+    configurable: true,
+    widgetType: "todoist",
+    category: "productivity",
+    initialHeight: 3,
+    initialWidth: 4,
+    initialOptions: {},
+    initialMeta: { updateCycle: { hours: 24 } }
```

Comment:
This file is autogenerated. Please run `yarn scan-widgets` after any change to `properties.ts`. Currently the values are ouf of sync.
I've looked into ways of improving this workflow by using babel macros, but without ejecting from create-react-app (or forking it), it doesn't get much better.